### ISSUE TITLE
Permutation importance show_weights unreadable in Jupyter dark

### ISSUE THREAD

**Issue by dwsmith1983** [eli5-org/eli5#13]
When the Jupyter theme is set to dark, it is impossible to see the the results from show_weights. Can we have the ability to set a dark or light switch so this can be useable no matter what theme someone is using?

I am using eli5 0.11.0

<img width="293" alt="Screen Shot 2021-04-25 at 1 52 55 PM" src="https://user-images.githubusercontent.com/7563201/115983778-8c8f1c80-a5cd-11eb-84b7-4244799d418c.png">

**Comment by TheWillyM4AK** [comment]
Im having the same problem.
